include/seqan3/range/container/concept.hpp

```diff
@@ -42,6 +42,50 @@
 
 #include <initializer_list>
 
+// remove if sequence_concept_modified_by_const_iterator_bug vanished from travis
+#include <string>
+
+namespace seqan3::detail
+{
+//!\privatesection
+
+/*!\interface seqan3::sequence_concept_modified_by_const_iterator <>
+ * \brief Can insert and erase used with const iterators
```

Comment:
Maybe: `Checks whether insert and erase can be used with const_iterator`.
And maybe also add an explantation why this was separated from the original concept definition,
and that it should go back in, if the travis bug is not of any concern anymore.